static/app/components/replays/table/replayTable.tsx

```diff
@@ -114,37 +100,13 @@ export default function ReplayTable({
   );
 }
 
-type TableProps = {
-  children: ReactNode;
+const ReplayTableWithColumns = styled(SimpleTable, {
+  shouldForwardProp: prop => prop !== 'columns',
+})<{
```

Comment:
Instead of forwarding this, could we make `grid-template-columns` an inline style? it's an [emotion best practice](https://emotion.sh/docs/best-practices#use-the-style-prop-for-dynamic-styles)

Reply:
inLInE stYLEs?1>?!!